pkg/secureframe/graphql.go

```diff
@@ -77,6 +77,8 @@ type Person struct {
 	InvitedAt                           time.Time `json:"invitedAt"`
 	ManagedComputer                     bool      `json:"managedComputer"`
 	Name                                string    `json:"name"`
+	OnboardingTasksDeadlineExceeded     bool      `json:"onboardingTasksDeadlineExceeded"`
```

Comment:
These are additional fields I sourced from the `graphql` response in the `Personnel` page.